### PR TITLE
Some minor improvements (rename 'core library' into 'MAVSDK-C++', etc.)

### PR DESCRIPTION
**Pull request by JonasVautherin**
I think we have converged to saying "MAVSDK-C++", "MAVSDK-Python", etc, and we don't really call it "core library" anymore. "core" is a part of MAVSDK-C++ (which is made of core + plugins), but the whole thing is MAVSDK-C++.

Those are minor changes, but that's also for me to get used to gitbook :innocent:.

### CODE REVIEW

File: en/README.md [hunk 1]
```diff
@@ -12,22 +12,22 @@ The library provides a simple API for managing one or more vehicles, providing p
 
 The library can run on a vehicle-based companion computer or on a ground-based GCS or mobile device (these devices have significantly more processing power that an ordinary flight controller, enabling tasks like computer vision, obstacle avoidance, and route planning).
 
-Developers can extend the core C++ SDK using plugins in order to add any other required MAVLink API (for example, to integrate a flight controller with custom cameras, gimbals, or other hardware over MAVLink).
+Developers can extend MAVSDK using plugins in order to add any other required MAVLink API (for example, to integrate a flight controller with custom cameras, gimbals, or other hardware over MAVLink).
```
Comment (hamishwillee) [inline en/README.md:15]: It probably is worth still making clear that the plugins are C++ not Python or whatever.

> extend MAVSDK using C++ plugins

Comment (JonasVautherin) [reply en/README.md:15]:
```suggestion
Developers can extend MAVSDK using C++ plugins in order to add any other required MAVLink API (for example, to integrate a flight controller with custom cameras, gimbals, or other hardware over MAVLink).
```

**Comment by hamishwillee** [comment]
Awesome. Great to see all these docs updates!

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Instructions how to setup a Gitbook toolchain and build this documentation can b
 ## License
 
 This documentation is licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
-The SDK itself is licensed under the permissive [BSD 3-clause](https://github.com/dronecore/DroneCore/blob/master/LICENSE.md).
+The SDK itself is licensed under the permissive [BSD 3-clause](https://github.com/mavlink/MAVSDK/blob/master/LICENSE.md).
```
Comment (julianoes) [inline README.md:19]: 😄

**Review by julianoes — COMMENTED** [review]
Thanks!

**Pull request merged**